Jenkinsfile.deploy

```diff
@@ -145,29 +145,27 @@ pipeline {
         stage('Build Vehicle Routing') {
             steps {
                 script {
-                    getOptawebVehicleRoutingMavenCommand().skipTests(params.SKIP_TESTS).run('clean install')
+                    buildOptaweb(getOptawebVehicleRoutingMavenCommand())
                 }
             }
             post {
                 always {
                     saveReports(params.SKIP_TESTS)
-                    // TODO: Enable together with the profile "integration-tests" once dockerhub pull limit is resolved.
-                    // archiveArtifacts(allowEmptyArchive: true, artifacts: "**/cypress/screenshots/**,**/cypress/videos/**", fingerprint: false)
+                    saveArtifacts(params.SKIP_INTEGRATION_TESTS)
```

Comment:
can we rename `saveReports` and `saveArtifacts` to have more explicit names so we know here what they are for ?

Reply:
Fixed fbe98eae273d36155a42686095df8a280a06df66